app.js

```diff
@@ -31,13 +31,24 @@ const favicon             = require('serve-favicon');
 const swaggerUi = require('swagger-ui-express');
 const swaggerDocument = require('./swagger.json');
 
+const helmet = require('helmet');
+const RateLimit = require('express-rate-limit')
+
 /* ------------------------------------------------------------------ *
                             API CONFIG
  * ------------------------------------------------------------------ */
 
 // define and configure express
 const app = express();
 const port = process.env.PORT || 3001;
+const limiter = new RateLimit({
+  windowMs: parseInt(process.env.WINDOW_MS, 10),
```

Comment:
Can we do defaults in case the environment variables aren't set?

Reply:
Yep.